packages/schema-poller/src/testConnection.ts

```diff
@@ -0,0 +1,18 @@
+import { Config, Driver } from 'neo4j-driver';
+import { initializeDriver } from './driverInitializer';
+
+export async function testConnectionTransiently(
```

Comment:
Why do we need this if the plugin is already telling you it fails to connect to neo4j when trying to save a connection which credentials don't work?

Reply:
yes, I can switch to doing that

Reply:
I've been looking into this - I can see that we're logging a console error in the schema poller, the persistent connection method will then retry;

`Unable to connect to Neo4j: Neo4jError: The client is unauthorized due to authentication failure.. Retrying in 30 seconds.`

are you happy for the schema poller to abort the retry in the event of an authorization failure, return this back to the language server and, in turn, the language server to notify the extension that authorization has failed?

unless I'm missing something 😅